### DISCUSSION TITLE
关于训练数据集大小及资源消耗

### DISCUSSION THREAD

**Discussion by Iriya99** [Equim-chan/Mortal#88] [Q&A]
想请问下作者训练时用到多少牌谱数据，训了多少epoch，以及各阶段需要用到多少gpu呢？另外有公开4.1代码的计划吗？

**Comment by hyskylord** [comment]
作者公布过v1的数据，印象中是10~20M个半庄。后面的版本似乎没有公开过。我个人比较希望了解的是离线训练的牌谱如何获取，考虑到天凤已经禁止Mortal使用其牌谱训练。

**Reply by HALF111** [reply]
您好，想咨询一下v1的数据在哪里能够下载到呢

**Reply by Iriya99** [reply]
> 作者公布过v1的数据，印象中是10~20M个半庄。后面的版本似乎没有公开过。我个人比较希望了解的是离线训练的牌谱如何获取，考虑到天凤已经禁止Mortal使用其牌谱训练。

训练需要几卡呀？在线强化学习需要额外的显卡/机器吗？

**Reply by fight5566jay** [reply]
我第一次聽到天鳳官方禁止使用牌譜訓練這件事
想請問官方把這個情報發布在什麼地方?
感謝

**Reply by Apricot-S** [reply]
> 我第一次聽到天鳳官方禁止使用牌譜訓練這件事 想請問官方把這個情報發布在什麼地方? 感謝

See <https://tenhou.net/sc/raw/>.

> ※天鳳と競合する製品への開発・応用を目的として牌譜を使用していただくことはできません。
※天鳳の牌譜は、天鳳での対戦を公正に楽しんでいただく目的で公開されています。天鳳での対戦を必要としないサービスへの応用は無償有償ともに行えません。一般の麻雀への応用を目的に牌譜を使用する場合は support@c-egg.com までお問い合わせください。

**Reply by fight5566jay** [reply]
@Apricot-S 感謝情報
看起來官方不是直接寫說牌譜禁止使用於模型訓練
我知道天鳳禁止未經認可的bot在其平台上進行對戰，但製作牌譜分析工具時使用其牌譜是否也在被禁止的範圍內?

**Reply by Apricot-S** [reply]
@fight5566jay
You are right that the rules do not explicitly say “training is prohibited.”
However, since they state that applications outside of Tenhou itself are not allowed, using the logs for analysis tools or other external services would naturally fall under this prohibited scope.

On the other hand, whether purely private, non‑public experiments might be tolerated is not clearly specified in the rules, so that remains a gray area.

**Note**: This is only a general interpretation of the wording in the rules, not legal advice. For a definitive answer, you should contact the Tenhou administrators directly

**Comment by EasilyGoDie** [comment]
> 作者公布过v1的数据，印象中是10~20M个半庄。后面的版本似乎没有公开过。我个人比较希望了解的是离线训练的牌谱如何获取，考虑到天凤已经禁止Mortal使用其牌谱训练。

用雀魂的
